projects/documentation/src/components/search-index.ts

```diff
@@ -72,7 +74,7 @@ export async function search(value: string): Promise<ResultGroup[]> {
             catagoryData.results.push({
                 name,
                 label: label(name),
-                url: item.ref,
+                url: BASE_URL + item.ref,
```

Comment:
This work was previously being managed [here](https://github.com/adobe/spectrum-web-components/blob/main/projects/documentation/scripts/build-search-index.js#L58-L61) and [here](https://github.com/adobe/spectrum-web-components/blob/main/projects/documentation/scripts/build-search-index.js#L77-L80). I'd definitely like to continue to process this at index build time, rather than site build time. This would likely involve a deeper understanding of how we leveraging our scripting in association with [wireit](https://github.com/google/wireit) to build and run the docs site. Something along the lines of breaking `"build:search-index"` out of the `"build:assets"` script and running a context specific version in each of `"build"`/`"serve"`, which will likely be the same for the two contexts, and `"build:production"`, which would need a value set for `process.eng.SWC_DIR` or possibly update the underlying script to accept a flag in the CLI instead...